src/core_plugins/vega/public/vega_view/vega_base_view.js

```diff
@@ -232,7 +239,10 @@ export class VegaBaseView {
       const handlerFunc = vegaFunctions[funcName];
       if (!handlerFunc || !this[handlerFunc]) {
         // in case functions don't match the list above
-        throw new Error(`${funcName}() is not defined for this graph`);
+        throw new Error(i18n.translate('vega.vegaParser.baseView.functionIsNotDefinedForGraphErrorMessage', {
+          defaultMessage: '{funcName}() is not defined for this graph',
```

Comment:
I would suggest pulling the parentheses into the `funcName` param too, since I assume that might otherwise be confusing for translators.